### PR TITLE
fix: remove normalized spec from schema

### LINKED ISSUE #6823: Schema shouldn't include normalized spec / FieldName
Somehow it currently does. 

Search for `Normalized` in https://raw.githubusercontent.com/vega/vega-lite/master/build/vega-lite-schema.json

### PR DESCRIPTION
**Pull request by domoritz**
fixes #6823

Needs https://github.com/vega/ts-json-schema-generator/pull/549

### CODE REVIEW

**Comment by domoritz** [comment]
Also addresses https://github.com/vega/vega-lite/commit/5c0b29951c901758c51701b458995d97fd85c92d#r41899248

File: src/compositemark/index.ts [hunk 1]
```diff
@@ -53,20 +53,18 @@ export function remove(mark: string) {
   delete compositeMarkRegistry[mark];
 }
 
-export type CompositeEncoding<F extends Field = Field> = Encoding<F> & ErrorExtraEncoding<F>;
+export type CompositeEncoding<F extends Field> = Encoding<F> & ErrorExtraEncoding<F>;
```
Comment (kanitw) [inline src/compositemark/index.ts:56]: Can you explain why do we remove the default `= Field` here?

Comment (domoritz) [reply src/compositemark/index.ts:56]: I found it less confusing to explicitly say the type. It makes me more confident that we didn't miss using string somewhere.

Comment (domoritz) [reply src/compositemark/index.ts:56]: Are you okay with this?

**Pull request merged**